PublicMapping/districtbuilder
# Add submission date and plan score url to organization map CSV export

## Overview

This PR adds the `submittedDt` and `planScoreUrl` columns from `project` to the organization map CSV export.

### Checklist


## Testing Instructions

- `./scripts/update`
- `./scripts/server`
- Log in -> My organizations -> Featured maps -> View all -> Press the download button and choose to "Download maps" 
- In the downloaded CSV, make sure the `submittedDt` and `planScoreUrl` columns are added

Closes #1183
